src/components/atoms/button/button.scss

```
@@ -0,0 +1,31 @@
+@use "../../variables" as *;
+
+.atoms-button{
+    width: 200px;
+    height: 55px;
+    color: $white;
+    background-color: $black;
+    font-size: $fontsize-s;
+    line-height: 20px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: 0.2s ease;
+}
+
+@media(max-width: $breakpoint-lg){
+    width: 150px;
+    height: 55px;
+}
+
+@media(max-width: $breakpoint-md){
+    width: 180px;
+    height: 45px;
+}
+
+
+&:hover{
+    color: $white;
+    background-color: $orange_bistre;
+}
```

```suggestion
  
  @media(max-width: $breakpoint-lg){
      width: 150px;
      height: 55px;
  }
  
  @media(max-width: $breakpoint-md){
      width: 180px;
      height: 45px;
  }
  
  
  &:hover{
      color: $white;
      background-color: $orange_bistre;
  }
}
```
chromaticのエラーは単に文法エラーっぽいです！
このようにメディアクエリや hover 擬似クラスについての記述も、 `.atoms-button` の括弧の中に入れるようにしてみてください！
scssの記法については、[課題ページ](https://shinonome.io/frontend-basic/lastAssignment/storybook_sass/)の "Sassとは" というセクションで軽く触れているので、確認してみてください 🍛